server/konnectors/maif.js

```diff
@@ -206,14 +208,10 @@ function getData(token, callback) {
 */
 function refreshToken(requiredFields, entries, data, next) {
   MaifUser.first((err, maifUser) => {
-    if (err) {
```

Comment:
I removed it because here, we use the err as a signal that the konnector is not configured yet ; and that we need to go fetch the token. 
This is badly designed, but didn't want to rewrite all the code from maif in a hurry, and without them.